cmd/gindoid/dataset.go

```diff
@@ -121,7 +121,8 @@ func cloneAndZip(repopath string, jobname string, targetpath string, conf *Confi
 
 	// Zip
 	log.Infof("Preparing zip file for %s", jobname)
-	zipfilename := filepath.Join(targetpath, jobname+".zip")
+	// use repository name for zip filename
+	zipfilename := filepath.Join(targetpath, reponame+".zip")
```

Comment:
Just for completeness sake: it might happen, that two different registered data sets from different organisations might have the same repository name and in turn identically named zip files.

Reply:
Yes that's possible now.  I guess it might be an issue if someone were to download all zips from registered datasets to the same location.  Alternatively, we could name the zip files based on the DOI and replace all `/` with `-`.